websites/W/Wolfy/presence.ts

```diff
@@ -1,10 +1,7 @@
 const presence = new Presence({
-		clientId: "721798189111443537"
-	}),
-	strings = presence.getStrings({
-		play: "presence.playback.playing",
-		browsing: "presence.activity.browsing"
-	});
+	clientId: "501842028569559061" // Official Wolfy Discord App Client ID, owned by Wolfy's Admin
```

Comment:
Have you asked the author about this?

Reply:
hmmm, that's something that i completely forgot because it's was just a presence request at first #286.

@fr0r is the original author, i can't contact him in DMs on Discord about that, i will try to ping on the PreMid server since he isn't on Wolfy's Discord